### ISSUE TITLE
Forms inside table cells don't adequately respond to the viewport

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar#1386]
**Describe the bug**
We use a 'table form' pattern in various places, the content inside the table cell doesn't appropriately respond to narrow viewports and represents a failure to comply to 1.4.10 Reflow at Level AA.

Admittedly this is a [fairly recent clarification](https://github.com/w3c/wcag/issues/1129) of the 1.4.10 guidelines and an updated paragraph was added in July 2020 ([meeting minutes](https://www.w3.org/2020/07/21-ag-minutes.html#item09))

![Screenshot 2021-03-05 at 09 14 42](https://user-images.githubusercontent.com/18653/110094410-b416f500-7d93-11eb-9bcb-4bf868a0c4e7.png)

**To Reproduce**
1. Go to a UI which uses a table-form pattern (e.g. XFP Form Templates (XFP-XFO-192))
2. Open the table form
3. Reduce the viewport to 320px
4. Observe the form breaking out of the container horizontally, requiring scrolling in two dimensions

**Expected behavior**
While the table is allowed to scroll in two dimensions, the content within the cell should respond so two-dimensional scrolling is not required.

**Screenshots**
![Screenshot 2021-03-04 at 16 59 09](https://user-images.githubusercontent.com/18653/110103452-e2013700-7d9d-11eb-878f-e8f60d0f9248.png)

**Additional context**
Relates to Platform Accessibility Statement item 9.

**Comment by Stanton** [comment]
Not present in Purple
